src/utils_project.js

```diff
@@ -728,29 +728,15 @@ go.utils_project = {
 
 // SUBSCRIPTION HELPERS
 
-    get_subscription_messageset_through_identity: function(im, mother_id, household_id) {
+    get_subscription_messageset_through_identity: function(im, mother_id) {
```

Comment:
A search of the repo showed that this method is never called with the `household_id` arg